harness/determined/cli/experiment.py

```diff
@@ -578,6 +578,30 @@ def experiment_logs(args: Namespace) -> None:
         )
 
 
+def set_log_retention(args: Namespace) -> None:
+    if not args.forever and not args.days:
```

Comment:
Possible bug here:
`det e set log-retention 1 --days 0` will fail, since `not 0` is True.

I'd change to `if not args.forever and not isinstance(args.days,int)`
Maybe even check that `args.days >= 0 ` after the fact.

Reply:
qq to @salonig23, does CLI accept `--days -1`? If no, we probably follow the same in webUI as well for the consistency .

Reply:
Talked to Haran, consensus is that we allow the user to manually input -1 in both CLI and UI

Reply:
fixed!